src/components/AppContentListItem/AppContentListItem.vue

```diff
@@ -0,0 +1,57 @@
+<!--
+ - @copyright Copyright (c) 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+	<router-link class="app-content-list-item" :to="to">
```

Comment:
We might have different uses than vue-router?

Reply:
So? What is your suggestion? Having a div as root element and optional link inside? Or let the app add the link?

Reply:
Either two props with :to and :href, or two components.
`AppContentListItemRouter.vue` `AppContentListItemLink.vue` ?

Reply:
let's add that in a follow-up PR.